sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -2236,3 +2238,21 @@ def test_upsert_entity_with_invalid_key_type(self, tables_storage_account_name,
                 self.table.upsert_entity(entity2)
         finally:
             self._tear_down()
+
+    @tables_decorator
+    @recorded_by_proxy
+    def test_list_tables_with_invalid_credential(self, tables_storage_account_name, tables_primary_storage_account_key):
```

Comment:
This is a (very) recent development, but per https://github.com/Azure/azure-sdk-for-python/pull/24799, the test proxy is getting an update that will require test arguments to be passed via `**kwargs` instead of with positional parameters. So this should get updated to:
```suggestion
    def test_list_tables_with_invalid_credential(self, **kwargs):
        tables_storage_account_name = kwargs.pop("tables_storage_account_name")
        tables_primary_storage_account_key = kwargs.pop("tables_primary_storage_account_key")
```
This applies to the async test as well

Reply:
Change of plans on the above -- you can leave the positional parameters as they are 🙂